gnuradio-runtime/include/gnuradio/sincos.h

```diff
@@ -12,12 +12,48 @@
 #define INCLUDED_GR_SINCOS_H
 
 #include <gnuradio/api.h>
+#include <math.h>
 
 namespace gr {
 
-// compute sine and cosine at the same time
-GR_RUNTIME_API void sincos(double x, double* sin, double* cos);
-GR_RUNTIME_API void sincosf(float x, float* sin, float* cos);
+#if defined(HAVE_SINCOS)
```

Comment:
Does it make sense to change the API on master by simplifying `sincos*` to just `template < T > sincos (const T x, T* sinx, T* cosx)` or better yet `template < T > sincos (const T x, T& sinx, T& cosx)` ... I mean, we're using C++ here which can handle functions with different arguments!

Reply:
for another PR